synfig-studio/src/gui/workarea.cpp

```diff
@@ -391,28 +390,21 @@ WorkArea::save_meta_data()
 	{
 		String data;
 		GuideList::const_iterator iter;
-		for(iter=get_guide_list_x().begin();iter!=get_guide_list_x().end();++iter)
-		{
+		for (iter = get_guide_list().begin(); iter != get_guide_list().end(); ++iter){
 			if(!data.empty())
 				data+=' ';
-			data+=strprintf("%f",*iter);
+			data+=strprintf("%f", (*iter).point[0]);
+			data+='*';
+			data+=strprintf("%f", (*iter).point[1]);
+			data+='*';
+			data+=strprintf("%f", (*iter).angle.get());
+			data+='*';
+			data+=strprintf("%d", (*iter).isVertical);
```

Comment:
You can make it into a single line! `"%f*%f` ...

Reply:
done